cv/cv/src/cv_utils/height_map.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+from typing import Dict, Tuple
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+HEIGHT: int = 180
+WIDTH: int = 330
+RAW_DEPTH_VALS_PATH: str = (
```

Comment:
Nit: consider using arg parser instead and using this as a default value but this is fine as is (since I don't think we'd need to generate a depth in a different location)